packages/tradershub/src/components/Tooltip/Tooltip.tsx

```diff
@@ -33,8 +33,8 @@ type TTooltipProps = {
  */
 const Tooltip = ({ alignment = 'bottom', children, className, message }: TTooltipProps) => {
     return (
-        <div className='relative w-max h-max group z-1'>
-            <div className='border rounded-md border-neutral-12'>{children}</div>
+        <div className='relative w-max h-max z-1'>
+            <div className='border rounded-md border-neutral-12 peer'>{children}</div>
```

Comment:
@hasan-deriv Are we using `peer` in any sibling classes or it's only here? Just wanna know how this is fixing issues here just by replacing group with peer 🙏

Reply:
Parent element has more space to interaction. But, it doesn't need that much of space. Only hover on element should trigger the tooltip.